package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "express": "^4.15.4",
     "react": "^15.6.1",
+    "react-buttons": "0.0.11",
```

Comment:
Unused dependency, you can run `yarn remove react-buttons` to remove this from package.json and update yarn.lock